src/shared-components/selectorButton/style.ts

```diff
@@ -104,7 +104,8 @@ const Selector = styled.div<{
   }
 `;
 
-const TextContainer = styled.p`
+const TextContainer = styled.div`
+  flex-grow: 1;
```

Comment:
Is this preventing a regression within this component library, or something for convenience in userland?

Reply:
I _thought_ it was preventing a regression when changing DOM elements, but it turns out I was just [breaking a nested CSS selector](https://github.com/curology/PocketDerm/blob/master/resources/assets/jsx/components/patient/setCustomization/selectors/moisturizerSelector/style.ts#L58-L60). I'll remove the `flex-grow` rule, as that particular styling is currently achieved via the consumer of the component.

Reply:
We can also add a className to that inner div (or some other easily identifiable token) so that nested selectors like that in user land are less brittle. Sorry 😞